Upstream-Status: Inappropriate [embedded specific]

Manipulating LD_LIBRARY_PATH can totally break builds when cross compiling.
Since we never run compiled binaries for the target, we should *never* need
to change LD_LIBRARY_PATH.

RP 9/6/2010

Index: curl-7.20.0/configure.ac
===================================================================
--- curl-7.20.0.orig/configure.ac	2010-06-09 11:06:30.000000000 +0100
+++ curl-7.20.0/configure.ac	2010-06-09 11:04:25.000000000 +0100
@@ -1461,9 +1461,10 @@
        dnl linker doesn't search through, we need to add it to LD_LIBRARY_PATH
        dnl to prevent further configure tests to fail due to this
 
-       LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$LIB_OPENSSL"
-       export LD_LIBRARY_PATH
-       AC_MSG_NOTICE([Added $LIB_OPENSSL to LD_LIBRARY_PATH])
+       #LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$LIB_OPENSSL"
+       #export LD_LIBRARY_PATH
+       #AC_MSG_NOTICE([Added $LIB_OPENSSL to LD_LIBRARY_PATH])
+       :
     fi
   fi
 
@@ -1684,9 +1686,10 @@
           dnl LD_LIBRARY_PATH to prevent further configure tests to fail
           dnl due to this
 
-          LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$gtlslib"
-          export LD_LIBRARY_PATH
-          AC_MSG_NOTICE([Added $gtlslib to LD_LIBRARY_PATH])
+          #LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$gtlslib"
+          #export LD_LIBRARY_PATH
+          #AC_MSG_NOTICE([Added $gtlslib to LD_LIBRARY_PATH])
+          :
         fi
       fi
 
@@ -1696,6 +1699,8 @@
 
 fi dnl OPENSSL != 1
 
+AC_SUBST(GNUTLS_REQUIRED)
+
 dnl ----------------------------------------------------
 dnl NSS. Only check if GnuTLS and OpenSSL are not enabled
 dnl ----------------------------------------------------
@@ -1781,9 +1786,10 @@
         dnl LD_LIBRARY_PATH to prevent further configure tests to fail
         dnl due to this
 
-        LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$nssprefix/lib$libsuff"
-        export LD_LIBRARY_PATH
-        AC_MSG_NOTICE([Added $nssprefix/lib$libsuff to LD_LIBRARY_PATH])
+        #LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$nssprefix/lib$libsuff"
+        #export LD_LIBRARY_PATH
+        #AC_MSG_NOTICE([Added $nssprefix/lib$libsuff to LD_LIBRARY_PATH])
+        :
       fi
 
     fi
@@ -1881,9 +1887,9 @@
        dnl libssh2_version is a post 1.0 addition
        AC_CHECK_FUNCS( libssh2_version )
 
-       LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$DIR_SSH2"
-       export LD_LIBRARY_PATH
-       AC_MSG_NOTICE([Added $DIR_SSH2 to LD_LIBRARY_PATH])
+       #LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$DIR_SSH2"
+       #export LD_LIBRARY_PATH
+       #AC_MSG_NOTICE([Added $DIR_SSH2 to LD_LIBRARY_PATH])
     fi
   else
     dnl no libssh2, revert back to clean variables
